Unreal/CarlaUnreal/Plugins/Carla/Source/Carla/Util/SensorSpawnerActor.cpp

```diff
@@ -270,3 +285,35 @@ void ASensorSpawnerActor::AttachSensorToActor(AActor* SensorActor)
     }
   }
 }
+
+void ASensorSpawnerActor::DrawRadarSensorPoints() const
+{
+  float velocity_range = VelocityRange; // m/s
+  for(const ASensor* CurrentSensor : SpawnedSensorsArray)
+  {
+    if(const ARadar* RadarSensor = Cast<ARadar>(CurrentSensor))
```

Comment:
```suggestion
    if(const ARadar* RadarSensor = Cast<ARadar>(CurrentSensor); CurrentSensor)
```
Apply the suggested change to check if `CurrentSensor` is null.
See the next documentation to understand why:
https://www.tutorialspoint.com/cplusplus17-if-statement-with-initializer

Reply:
At the end of the day is the same, with the initializacion inside of the if statment, if the result of the operation is nullptr, AKA Current sensor is not a Radar, the code inside if will be elipsed